Dockerfile

```diff
@@ -1,30 +1,71 @@
-FROM haskell:8.6 AS build
-USER root
-# TODO: install Yices, too (and CVC4?, Boolector?)
+from debian:buster AS solvers
```

Comment:
I'm surprised docker allows lowercase `from`. It's "technically" a syntax error in the file format.

Reply:
I don't know how that got there. Good catch!